website/content/docs/plugins/index.mdx

```diff
@@ -0,0 +1,200 @@
+
+## How Plugins Work
+
+Packer plugins are completely separate, standalone applications that the core
+of Packer starts and communicates with.
+
+These plugin applications aren't meant to be run manually. Instead, Packer core
+executes them as a sub-process, run as a sub-command (`packer plugin`) and
+communicates with them. For example, the Shell provisioner is actually run as
+`packer plugin packer-provisioner-shell`. The next time you run a Packer build,
+look at your process list and you should see a handful of `packer-` prefixed
+applications running.
+
+
+<Tabs>
+<Tab heading="Plugins installed with Packer init (from Packer v1.7.0)">
+
+~> **Note**: Only _multi-plugin binaries_ -- that is plugins named
+packer-plugin-*, like the `packer-plugin-amazon` -- are expected to work with
+Packer init. The legacy `builder`, `post-processor` and `provisioner` plugin
+types will keep on being detected but Packer cannot install them automatically.
+
+## Installing Plugins
+
+1. Add a
+[`required_plugin`](/docs/templates/hcl_templates/blocks/packer#specifying-plugin-requirements)
+block to your `packer` block, each block will tell Packer what version(s) can be
+installed, make sure to set a correct [version constraint
+string](/docs/templates/hcl_templates/blocks/packer#version-constraints). 
+  Example `required_plugins` block :
+
+  ```hcl
+  packer {
+    required_plugins {
+      myawesomecloud = {
+        version = ">= 2.7.0"
+        source = "azr/myawesomecloud"
+      }
+      otherawesomecloud = ">= 2.7.0"
+    }
+  }
+  ```
+
+2. Run [`packer init`](/docs/commands/init) to install all missing plugin binaries.
+
+## Names and Addresses
+Each plugin has two identifiers:
+
+* A `source` address, which is only used when requiring a plugin.
+* A unique **local name**, which is used everywhere else in a Packer 
+  configuration.
+
+### Local Names
+
+Local names allow you to access the components of a plugin and must be unique
+per configuration.
+
+Here using the previous example if the required plugin `azr/myawesomecloud`
+defined an `ebs` builder, this builder will be made available by using a
+`myawesomecloud-ebs` source block :
+
+```hcl
+source "myawesomecloud-ebs" "example" {...}
+```
+
+## Source Addresses
+
+A plugin's source address is its global identifier. It also specifies the
+primary location where Packer can download it.
+
+Source addresses consist of three parts delimited by slashes (`/`), as
+follows:
+
+`[<HOSTNAME>/]<NAMESPACE>/<TYPE>`
+
+* **Hostname** (optional): The hostname of the location/service that
+  distributes the plugin. If omitted, this defaults to
+  `github.com`, we recommend explicitly setting the hostname.
+
+* **Namespace:** An organizational namespace within the specified host.
+  This often is the organization that publishes the plugin. If omitted, this
+  defaults to `hashicorp`, we recommend explicitly setting the namespace.
+
+* **Type:** A short name for the platform or system the plugin manages. The
```

Comment:
Should an example be added to make it more clear. Like 
githu.com/hashicorp/packer-plugin-amazon source would be github.com/hashicorp/amazon

Reply:
Yes, there's one just down there.

Reply:
Oh yeah! I saw it after commenting on this. Looks great btw